test/OverflowTable.test.js

```diff
@@ -315,7 +315,7 @@ describe("OverflowTable", () => {
 				} catch (error) {
 					done(error);
 				}
-			}, 150);
+			}, 1000);
```

Comment:
One second is a long time to wait 😅 would a lower value do the trick?
 
```suggestion
			}, 350);
```